lib/ansible/modules/storage/netapp/na_ontap_nfs.py

```diff
@@ -44,6 +44,10 @@
     - status of if NFSv3 clients see change in FSID as they traverse filesystems.
     choices: ['enabled', 'disabled']
     version_added: '2.7'
+  nfsv4_fsid_change:
+    description:
+    - status of if NFSv4 clients see change in FSID as they traverse filesystems.
+    choices: ['enabled', 'disabled']
```

Comment:
For all the new fields, you need to add 
version_added: ‘2.9’

Maybe this could explain one of the shippable issues.